.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: php
+cache:
+  directories:
+    - vendor
+php:
+  - 5.6
+  - 7.0
+
+install:
+  - composer install -o --prefer-source --no-interaction
```

Comment:
Just `composer install --no-interaction` will be fine, `-o` could hide namespacing issues.